tools/kafka/loopback/README.md

```diff
@@ -0,0 +1,53 @@
+# Loopback service for 100K epic
```

Comment:
Please, try  to explain this service without refer to "100k epic".
What do you think about that @rascaraficci ?

Reply:
It makes sense!